packages/jest-runtime/src/index.ts

```diff
@@ -737,12 +740,70 @@ export default class Runtime {
     ) {
       return this.loadEsmModule(resolved, query);
     }
-
+    if (skipCjsModules) {
+      // @ts-expect-error - exiting
```

Comment:
can we add ` | undefined` to the return type instead? or `null` possibly to be more explicit

Reply:
I can do, not sure how cascading that will be though (e.g., TS errors on all the callsites) - it can only be null in this one case, which is only called from line 764, this was also the "established pattern" from line 594.